### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by jabajke**

### CODE REVIEW

File: authentication/views.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    queryset = User.objects.all()
+
+    def get(self, request, *args, **kwargs):
+        token = request.GET.get('token')
+        try:
+            payload = jwt.decode(token, settings.SECRET_KEY, algorithms=['HS256'])
+            serializer = self.get_serializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
+            user_id = payload['user_id']
+            serializer.save(user_id=user_id)
```
Comment (shprexen) [inline authentication/views.py:29]: you don't need to decode your token for receiving user_id from payload, as you can get user object from `request.user`

**Review by shprexen — CHANGES_REQUESTED** [review]
merge previous branch to dev and then merge merge dev to your current feature branch to prevent merge conflicts

File: authentication/views.py (hunk 1 above)
Comment (jabajke) [reply authentication/views.py:29]: @shprexen are u sure? so we dont have any user state using jwt, and i think that request.user would return anonymous

Comment (shprexen) [reply authentication/views.py:29]: Do you get the token from query params?

**Review by shprexen — APPROVED** [review]

**Pull request merged**